cbc-repeated-iv/src/cbc_game.rs

```diff
@@ -0,0 +1,575 @@
+                we can guess that data structure is different this time...".to_string(),
+                "
+    ▄▄▄▄▄▄▄▄▄▄   ▄            ▄▄▄▄▄▄▄▄▄▄▄  ▄▄▄▄▄▄▄▄▄▄▄  ▄    ▄         ▄         ▄     ▄▄▄▄▄▄▄▄▄
+   ▐░░░░░░░░░░▌ ▐░▌          ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░▌  ▐░▌       ▐░▌       ▐░▌   ▐░░░░░░░░░▌
+   ▐░█▀▀▀▀▀▀▀█░▌▐░▌          ▐░█▀▀▀▀▀▀▀█░▌▐░█▀▀▀▀▀▀▀▀▀ ▐░▌ ▐░▌       ▄█░█▄▄▄▄▄▄▄█░█▄ ▐░█░█▀▀▀▀▀█░▌
+   ▐░▌       ▐░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌▐░▌       ▐░░░░░░░░░░░░░░░▌▐░▌▐░▌    ▐░▌
+   ▐░█▄▄▄▄▄▄▄█░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌░▌         ▀█░█▀▀▀▀▀▀▀█░█▀ ▐░▌ ▐░▌   ▐░▌
+   ▐░░░░░░░░░░▌ ▐░▌          ▐░▌       ▐░▌▐░▌          ▐░░▌           ▐░▌       ▐░▌  ▐░▌  ▐░▌  ▐░▌
+   ▐░█▀▀▀▀▀▀▀█░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌░▌         ▄█░█▄▄▄▄▄▄▄█░█▄ ▐░▌   ▐░▌ ▐░▌
+   ▐░▌       ▐░▌▐░▌          ▐░▌       ▐░▌▐░▌          ▐░▌▐░▌       ▐░░░░░░░░░░░░░░░▌▐░▌    ▐░▌▐░▌
+   ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄▄▄ ▐░█▄▄▄▄▄▄▄█░▌▐░█▄▄▄▄▄▄▄▄▄ ▐░▌ ▐░▌       ▀█░█▀▀▀▀▀▀▀█░█▀ ▐░█▄▄▄▄▄█░█░▌
+   ▐░░░░░░░░░░▌ ▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░░░░░░░░░░░▌▐░▌  ▐░▌       ▐░▌       ▐░▌   ▐░░░░░░░░░▌
+    ▀▀▀▀▀▀▀▀▀▀   ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀▀▀▀▀▀▀▀▀▀▀  ▀    ▀         ▀         ▀     ▀▀▀▀▀▀▀▀▀"
+                    .to_string()
+            ),
+            _ => (
+                "What are you doing here, step-bro??? UwU".to_string(),
+                "Okay, here's some small secret crypto-tip for you:".to_string(),
+                "https://www.youtube.com/watch?v=dQw4w9WgXcQ".to_string()
+            )
+        }
+    }
+
+    // Creating beautified ascii table
+    // User must send strings (lines), length of each line and padding
+    // We need to use old (3 years ago last commit) crate, because new one
+    // Can't handle Colorized strings properly
+    fn create_table(
+        title: &str,
+        data: Vec<Vec<&String>>,
+        padding: usize,
+    ) -> anyhow::Result<String> {
+        // At first let's create and build custom styles for title and main table
+        let main_table_format = FormatBuilder::new()
+            .column_separator('║')
+            .borders('║')
+            .separators(&[LinePosition::Top], LineSeparator::new('═', '╦', '╔', '╗'))
+            .separators(
+                &[LinePosition::Intern],
+                LineSeparator::new('═', '╬', '╠', '╣'),
+            )
+            .separators(
+                &[LinePosition::Bottom],
+                LineSeparator::new('═', '╩', '╚', '╝'),
+            )
+            .padding(padding, padding)
+            .build();
+
+        let title_table_format = FormatBuilder::new()
+            .column_separator('║')
+            .borders('║')
+            .separators(&[LinePosition::Top], LineSeparator::new('═', '╦', '╔', '╗'))
+            .separators(
+                &[LinePosition::Intern],
+                LineSeparator::new('═', '╬', '╠', '╣'),
+            )
+            .padding(padding, padding)
+            .build();
```

Comment:
What is the difference between these formats? They look almost the same :eyes:

Reply:
Yes, almost the same, except the title format does not have a bottom border.
I thought that it would be better to create a format for the title and then expand it with the bottom border for the main table. But style builder actually has method `build`, which I think - does not allow us to expand style. Also, `build` returns `TableFormat` instead of `TableFormatBuilder`.

But I think that I can save the builder into a mutable variable. And build those styles only when I would call `set_format` for my tables.